documentation/developer-guide/modules/tooling-guide/pages/java-aspect-tooling.adoc

```diff
@@ -19,6 +19,15 @@ an error occurs, the object can also provide the original exception. Compared wi
 as a return type instead has the advantage of enabling composable and better readable code. Please see the
 https://www.vavr.io/vavr-docs/[Vavr User Guide] for more information.
 
+[[versioning]]
+== Getting the right version
+
+The tooling can be made available in two different versions: `release` and `milestone`. The release version represents a stable version of the product
+and can be referenced in your projects in the usual way from the Maven Central repository. There is also the possibility to have the intermediate builds
+made available, these are called `milestone` builds. Instead of Maven Central, these are released via 
+https://github.com/orgs/OpenManufacturingPlatform/packages?repo_name=sds-sdk[GitHub Packages] mechanism.
+For more information about how to reference such artifacts in your projects, please refer to https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry#installing-a-package[Github - Installing a package].
```

Comment:
Although it is technically correct to link to Github's guide, this guide does not make it clear how to actually use the packages. In particular, it is not mentioned explicitly that you need to add a `<repository>`. I'd mention the following example section that can be added to the pom.xml (together with 1-2 sentences of additional explanation):
```xml
   <repositories>
      <repository>
         <id>github</id>
         <name>OpenManufacturingPlatform SDS-SDK</name>
         <url>https://maven.pkg.github.com/OpenManufacturingPlatform/sds-sdk</url>
         <releases><enabled>true</enabled></releases>
         <snapshots><enabled>true</enabled></snapshots>
      </repository>
   </repositories>
```